### ISSUE TITLE
UI Feedback

### ISSUE THREAD

**Issue by ssvillarosa** [PUP-BSIT/project-fusion#2]
- After you log in, which page the user will be redirected to? I believe it's either the "My Products" or the "My Sales" page. If so, what is the use of "Home" in the navigation? If the user clicks it, where should it go?
- In the left of MyAccount, there are two buttons, order and product. If I click the "Order" button, will it display to "My Sales" page? Would it be confusing if the two are named differently? Same applies to "Product" button. Be consistent in naming so your user won't get confuse in navigating your page.

**Comment by Cayiee** [comment]
Dear Professor Villarosa,

Thank you for your insightful feedback. In response, we have implemented changes to redirect users to the "Orders" page upon login. Furthermore, we have ensured naming consistency by revising "My Sales" to "Orders" and "My Products" to "Products."

The "Home" button has been removed to streamline the interface. We trust these adjustments will enhance the user experience.

Your guidance is highly appreciated.

Best regards,
Castillo, Krystel Caye R.

**Comment by Cayiee** [comment]
Dear Professor Villarosa,

I hope this message finds you well. I wanted to update you on the recent changes we've made based on your insightful feedback. We have successfully implemented adjustments to redirect users seamlessly to the "Orders" page upon login. Additionally, we've ensured naming consistency by updating "My Sales" to "Orders" and "My Products" to "Products."

In the interest of streamlining the interface, we have also removed the "Home" button. We believe these modifications will significantly contribute to enhancing the overall user experience. Your guidance has been invaluable throughout this process, and we are pleased to share that the implementation is now complete.

Thank you once again for your support and valuable feedback.

Best regards,
Castillo, Krystel Caye R.
